security/v1/types.go

```diff
@@ -183,6 +183,7 @@ var (
 	FSScaleIO                   FSType = "scaleIO"
 	FSStorageOS                 FSType = "storageOS"
 	FSTypeCSI                   FSType = "csi"
+	FSTypeEphemeral             FSType = "ephemeral"
```

Comment:
fun.  I'm not sure all the spots this ripples to.  @stlaz I'm betting at least to apiserver-library-go.

Reply:
Yes, I noticed that this change was necessary when a bumped apiserver-library-go resulted in failing unit tests. With this change and an associated a-l-g change the unit tests now pass.